test/e2e/windows/density.go

```diff
@@ -75,17 +74,12 @@ var _ = SIGDescribe("[Feature:Windows] Density [Serial] [Slow]", func() {
 type densityTest struct {
 	// number of pods
 	podsNr int
-	// number of background pods
-	bgPodsNr int
```

Comment:
cc @kubernetes/sig-scalability-pr-reviews @kubernetes/sig-windows-misc  on removal of these. was the intent for these types to remain as close as possible to the original types in test/e2e_node?

Reply:
@liggitt  these vars still remain?THX